.github/workflows/build.yml

```diff
@@ -1,6 +1,6 @@
 name: Build Application
 
-on: [push]
+on: [pull_request]
```

Comment:
need both

Reply:
Well, I thought you need to replace it only. But now it's fixed.